### ISSUE TITLE
Installing 3.4.7 results in ParseError

### ISSUE THREAD

**Issue by stphnmartin** [ethercreative/simplemap#143]
### Description
Installing Maps 3.4.7 via CP results in a ParseError. Process finishes the installation process, displays the success icon then this ParseError:

ParseError
syntax error, unexpected '=>' (T_DOUBLE_ARROW), expecting :: (T_PAAMAYIM_NEKUDOTAYIM)
↵
Caused by: ParseError
syntax error, unexpected '=>' (T_DOUBLE_ARROW), expecting :: (T_PAAMAYIM_NEKUDOTAYIM)
in ......./vendor/ether/simplemap/src/services/MapService.php at line 212

1. in ........../vendor/ether/simplemap/src/services/MapService.php at line 212
203204205206207208209210211212213214215216217218219220221                    			$location = GeoService::latLngFromAddress($location, $country);
        else if ($location instanceof Map)
            $location = ['lat' => $location->lat, 'lng' => $location->lng];
        else if (!is_array($location) || !isset($location['lat'], $location['lng']))
            $location = null;
 
        if ($location === null)
            return false;
 
        list('lat' => $lat, 'lng' => $lng) = $location;
 
        // Normalize radius
        if (!is_numeric($radius))
            $radius = (float) $radius;
 
        if (!is_numeric($radius))
            $radius = 50.0;
 
        // Normalize unit

### Steps to reproduce

1. Install maps from the Craft plugin store
2. View success icon
3. See ParseError display.

View screencast of install: https://www.screencast.com/t/ot3ioe5v2D7b

### Additional info

- Craft version: 3.1.19
- SimpleMap version: 3.4.7
- PHP version: 7.0.33
- Database driver & version: MySQL 5.7.25
- Other Plugins:

Contact Form	2.2.3
Contact Form Templates	1.0.1
Enupal Snapshot	1.2.3
Feed Me	3.1.16
Field Manager	2.0.5
Freeform Lite	2.5.15
Logout Redirect	2.0.1
MailChimp Lists	1.0.5
Redactor	2.3.2
SEOmatic	3.1.46
Store Hours	2.1.1.1
SuperSort	3.0.1
Super Table	2.1.16
Wordsmith	3.0.4

**Comment by engram-design** [comment]
Getting this as well. 

According to the PHP docs for [list()](https://www.php.net/manual/en/function.list.php):
```
7.1.0 | It is now possible to specify keys in list(). This enables destructuring of arrays with non-integer or non-sequential keys.
```

So I believe this is an incompatibility issue with PHP < 7.1. If you can update to PHP 7.1+ in the meantime, this should fix the issue

**Comment by Tam** [comment]
Fixed in 3.4.8
